mail.md

```diff
@@ -0,0 +1,35 @@
+**For the first time ever, Exercism is hiring!**
+Thanks to the generosity of our community and supporting organisations, we're about to take Exercism to a whole new level.
+ 
+---
+ 
+Hi exercism-bot,
+
+Since we started, we've been longing to get to a point where we can hire a full-time team to unleash Exercism's potential. 
+After the launch and success of the new platform at the end of last year, and generous support from our community and supporting organisations, we're finally there. 
+We couldn't be more excited! 🥳
+
+In hiring the team and growing Exercism, we have three main goals:
+
+- Firstly, we want to do more of what we've got right already - providing a fun way for developers to improve their skills and learn new languages. 
```

Comment:
```suggestion
- Firstly, we want to do more of what we have gotten right already - providing a fun way for developers to improve their skills and learn new languages. 
```

The contraction is OK, but perhaps clearer without, while we have gotten a some things right already.